kafka-connector/src/main/java/com/sequenceiq/cloudbreak/kafka/model/KafkaRecordRequest.java

```diff
@@ -8,7 +8,7 @@
     private final String key;
 
     private KafkaRecordRequest(Builder builder) {
```

Comment:
You have deleted Kudu. Can we delete Kafka and related libs? Smaller surface for CVEs...

Reply:
ok i have removed it, although it can be useful later for different purposes (like monitoring), but can be re-added once it's needed